packages/mdl-card/mdl-card.scss

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-typography/mixins";
+
+.mdl-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 0;
+  overflow: hidden;
+  box-sizing: border-box;
+
+  &__primary {
+    padding: 16px;
+
+    /* Add extra space before large title. */
+    &.mdl-card--large-title {
+      padding-top: 8px;
+    }
+
+    /* Add extra padding if this is the last block. */
+    &:last-child {
+      padding-bottom: 24px;
+    }
+  }
+
+  &__supporting-text {
+    padding: 8px 16px;
+
+    @include mdl-typography(body1);
+    @include mdl-typography-color(primary, true);
```

Comment:
We may want to add a TODO here to visit this once we have thematic text in place. We may want to invert the typographic colors based on the theme's "background color", e.g. whether or not we need a dark color.